zebra/zebra_nhg.h

```diff
@@ -0,0 +1,29 @@
+/* Zebra NHG header.
```

Comment:
Same question about spelling-out 'nexthop group' in some way, just for the benefit of future readers